src/pages/Admin/Charity/Dashboard/Balance.tsx

```diff
@@ -0,0 +1,46 @@
+import { PropsWithChildren } from "react";
+import { humanize } from "helpers";
+
+type Props = {
+  type: "Sustainability Fund" | "Donations";
+  current: number;
+  pending: number;
+  payedOut: number;
+};
+
+export default function Balance({ type, current, pending, payedOut }: Props) {
+  return (
+    <div className="@container rounded border border-prim bg-orange-l6 dark:bg-blue-d6 p-4">
+      <h4 className="uppercase text-sm @xs:text-xl font-bold mb-5">
+        {type} account
+      </h4>
+      <p className="text-xs @xs:text-sm text-gray-d1 dark:text-gray mb-8">
+        Note: balances are estimations.
```

Comment:
@misicnenad We can remove this copy. Balances are exact (for now).
```suggestion
```

Reply:
Done, see https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2480/commits/faa7ae5b1cfd81f6485ea64851be9d4dab2a7bb3